src/App.jsx

```diff
@@ -14,57 +15,71 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [isSelected, setIsSelected] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  const handleSelectGood = good => {
+    setIsSelected(good);
+  };
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+  const clearSection = () => {
+    setIsSelected('');
+  };
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {!isSelected ? 'No goods selected' : `${isSelected} is selected`}
+        {isSelected && (
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={clearSection}
+          />
+        )}
+      </h1>
 
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
+      <table className="table">
+        <tbody>
+          {goods.map(good => (
+            <tr
+              key={good}
+              data-cy="Good"
+              className={
+                isSelected === good ? 'has-background-success-light' : ''
+              }
             >
-              -
-            </button>
-          </td>
+              <td>
+                {isSelected !== good && (
+                  <button
+                    data-cy="AddButton"
+                    type="button"
+                    className="button"
+                    onClick={() => handleSelectGood(good)}
+                  >
+                    +
+                  </button>
+                )}
+                {isSelected === good && (
```

Comment:
You can create a variable for this checking `isSelected === good` and use it in two places